packages/docs/introduction.yaml

```diff
@@ -25,60 +25,57 @@ _ref:
-              - All apps use the same structured config schema, this makes it easy to debug large apps or pick up where others left off.
-              - Nothing is hidden in a GUI. This allows you to do basic essential stuff, like copy, paste, find, replace etc. which makes developing apps more productive.
-              - App config is just data, thus you can even develop scripts to create and manage your apps.
+            The Lowdefy schema is a simple definition of a web app which primarily makes use of block, action, operator and request definitions.
+
+            Some advantages of writing web apps in YAML:
+              - All apps use the same schema, this makes it easy to debug large apps or pick up where others left off.
+              - Nothing is hidden in a GUI. This allows you to do basic stuff, like copy, paste, find, replace etc. which makes developing apps more productive.
+              - App config is just data, thus you can develop scripts to create and manage your apps.
               - YAML files work with your favorite developer and source control tools.
 
             > If you aren't a fan of YAML, you can also build your app using JSON files.
 
-            ### Work with your data where it is stored
-
-            Lowdefy apps can connect to multiple data bases, APIs and apps. Day to day business requires people working with data which is stored in a variety of different data bases and applications.
-
-            Too often data is exported, duplicated and lost, creating additional risk. Lowdefy only runs your application config, and does not store any data, but rather connects to your data sources and APIs.
+            ## Build future proof web apps on an open-source platform
 
-            ### Build future proof web applications on an open-source platform
+            We've built Lowdefy to scale our own ability to develop and maintain web apps and internal tools for our customers.
 
-            Lowdefy is a open-source Apache-2 licensed project, free from vendor lock-in. Our business model is to build optional "nice-to-have" services best suited for Lowdefy applications, enriching the Lowdefy ecosystem.
+            Lowdefy is an open-source Apache-2 licensed project, free from vendor lock-in. We hope to see many community open-source Lowdefy apps and plugins in the future and actively nurture the Lowdefy community on [Github](https://github.com/lowdefy/lowdefy/discussions) and [Discord](https://discord.gg/QQY9eJ7A2D) to help other developers to get started with Lowdefy and grow the ecosystem.
 
-            Since Lowdefy apps are easy to manage in a git repository, we hope to see many community open-source Lowdefy apps in the future.
+            To date, more than 20 000 apps has been built using Lowdefy.
 
-            ### Deploy your Lowdefy app as you like
+            In the future plan to build optional "nice-to-have" services best suited for Lowdefy apps to further enriching the Lowdefy ecosystem.
```

Comment:
In the future **we** plan to build...

Reply:
Change made in 2185f7c393e3cb32cf216ae86ec4d59f0b1a25a3